packages/web-runtime/src/store/user.ts

```diff
@@ -46,6 +46,10 @@ const mutations = {
     state.language = user.language
     state.role = user.role
     sentrySetUser({ username: user.id })
+
+    if (user.quota) {
+      this.commit('SET_QUOTA', user.quota)
+    }
```

Comment:
Fixes problem: Navigate to spaces -> Reload page -> Quota not shown in user menu